lib/charms/layer/docker_registry.py

```diff
@@ -70,6 +70,15 @@ def configure_registry():
         if os.path.isfile(tls_ca):
             http['tls']['clientcas'] = [tls_ca]
             docker_volumes[tls_ca] = '/etc/docker/registry/ca.crt'
+
+    # debug https://docs.docker.com/registry/configuration/#debug
+    # The debug server always listens on port 5001, with debug-port used
```

Comment:
this link is a 404

Reply:
Fun, looks like the registry was recently donated to the CNCF (as noted at https://docs.docker.com/registry/). Would you like all the URLs updated in this PR, or in a separate one? I've changed this URL specifically since it's directly related to the changes.

Reply:
Ah... just the one is fine.